kml/serdes_gpx.cpp

```diff
@@ -210,6 +224,10 @@ void GpxParser::CharData(std::string value)
       else if (currTag == gpx::kDesc)
         m_categoryData->m_description[gpx::kDefaultLang] = value;
     }
+    if (currTag == gpx::kColor)
+    {
```

Comment:
We don't use braces for one-line ifs and loops.

Reply:
Fixed